nvi-test/src/main/java/no/sikt/nva/nvi/test/TestUtils.java

```diff
@@ -68,6 +70,10 @@ public static String randomYear() {
     return String.valueOf(randomIntBetween(START_DATE.getYear(), LocalDate.now().getYear()));
   }
 
+  public static String randomTitle() {
+    return String.format("%s %d", FAKER.book().title(), randomInteger(CURRENT_YEAR));
+  }
+
```

Comment:
Added because `FAKER` didn't have all that many unique book titles, making unrelated tests flaky